app/assets/javascripts/eventscripts/create_event.js

```diff
@@ -21,12 +21,65 @@ $( document ).on('turbolinks:load', function() {
         }
     });
 
-    $("#event_enddate").on("change", changeDur);
+    $("#event_enddate").on("change", adaptDuration);
 
+    $("#event_gameday_duration").on("change",durationChanged);
+
+    $("#event_game_mode").on("change",callChanged);
+
+    $("#event_max_teams").on("change",callChanged);
+
+    function callChanged()
+    {
+        adaptDuration();
+        durationChanged();
+    }
+    function durationChanged()
+    {
+
+        var start = $("#event_startdate").val();
+        var end = $("#event_enddate").val();
+        var system = $("#league_game_mode").val();
+        var gameday_dur =$("#league_gameday_duration").val();
+        var participants = $("#league_max_teams").val();
```

Comment:
You could extract some of this into `callChanged` since its the same in both `durationChanged` and `adaptDuration`